### DISCUSSION TITLE
How to bypass the anti-pistons

### DISCUSSION THREAD

**Discussion by cosplgay** [GriefPrevention/GriefPrevention#1555] [Q&A]
Hello,

In my server I am making a redstone contraption in a world guard region. I need there to be a piston, but I don't want to make a land claim inside of my region for one piston. Is there any way to bypass the fact that pistons wont move unless inside of claim without disabling the feature entirely?

**Comment by bigpresh** [comment]
Your choices are to set the [PistonMovement](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement) configuration setting to a value that will allow pistons to move outside of claims (allowing people to use them to grief, which you probably don't want) or just create a claim... if it were me I'd just do the latter and all is well :)
